staging/src/k8s.io/apiextensions-apiserver/test/integration/table_test.go

```diff
@@ -73,12 +96,14 @@ func newTableInstance(name string) *unstructured.Unstructured {
 				"gamma":   "bar",
 				"delta":   "hello",
 				"epsilon": []int64{1, 2, 3},
+				"zeta":    5,
 			},
 		},
 	}
 }
 
 func TestTableGet(t *testing.T) {
+	defer utilfeaturetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, apiextensionsfeatures.CustomResourceWebhookConversion, true)()
```

Comment:
do we have a test that colums are rejected if the feature gate is off?

Reply:
We test it in validation_test.go unit tests.